frontend/src/components/Trial/Trial.tsx

```diff
@@ -63,47 +63,15 @@ const Trial = (props: TrialProps) => {
     // Create result data
     const makeResult = useCallback(
         async (result: { type: 'time_passed' }) => {
+
             // Prevent multiple submissions
             if (submitted.current) {
                 return;
             }
-            submitted.current = true;
-
-            // TODO: Check if we can find another solution for
-            // the default value of form than [{}]
-            const form = feedback_form ? feedback_form.form : [{}];
-
-            if (result.type === "time_passed") {
-                form.map((formElement) => (formElement.value = "TIMEOUT"));
-            }
-
-            if (feedback_form) {
-
-                if (feedback_form.is_skippable) {
-                    form.map((formElement => (formElement.value = formElement.value || '')))
-                }
-
-                const breakRoundOn = config.break_round_on;
-                const shouldBreakRound = breakRoundOn && checkBreakRound(form.map((formElement) => formElement.value), breakRoundOn);
-                const shouldCallOnNextInOnResult = !shouldBreakRound
-
-                await onResult(
-                    {
-                        decision_time: getAndStoreDecisionTime(),
-                        form,
-                        config
-                    },
-                    false,
-                    // if we break the round, we don't want to call onNext in onResult
-                    shouldCallOnNextInOnResult
-                );
-
-                if (shouldBreakRound) {
-                    onNext(true);
-                }
 
-            } else {
+            submitted.current = true;
 
+            if (!feedback_form) {
 
                 if (result_id) {
```

Comment:
This bit is in here for "historical reasons" - the `Trial` has a `result_id` at top level, but this is not necessary anymore. (Before a refactor, the "breaking round" mechanism was handled through this top level result id.) I checked, and indeed, no `Trial` in the current rules files contains a `result_id`. We should remove `result_id` from the backend `Trial` constructor and this condition here. (Can be left for later though.)

Reply:
I've created an issue for this #1257 Can you expand upon it if necessary?